lib/generators/inertia/install/install_generator.rb

```diff
@@ -103,7 +103,7 @@ def install_inertia
                              before: '<%= vite_client_tag %>'
           end
 
-          gsub_file application_layout.to_s, /<title>/, '<title inertia>' unless svelte?
+          gsub_file application_layout.to_s, /<title>/, '<title inertia>'
```

Comment:
The Svelte component from the doc above needs the inertia tag to properly dedup so I removed this.